packages/manager/modules/octavia-load-balancer/src/create/controller.js

```diff
@@ -0,0 +1,10 @@
+import { PRODUCT_LINK } from './constants';
+
+export default class OctaviaLoadBalancerCreateCtrl {
+  $onInit() {
+    this.productPageLink =
+      PRODUCT_LINK[this.user.ovhSubsidiary] || PRODUCT_LINK.WE;
```

Comment:
```suggestion
      PRODUCT_LINK[this.user.ovhSubsidiary] || PRODUCT_LINK.DEFAULT;
```
You should create a `DEFAULT` entry to manage default value instead of `WE`